deepcell/model_zoo/panopticnet.py

```diff
@@ -299,11 +319,19 @@ def PanopticNet(backbone,
             backbone.  3 is the default for all current backbones.
         kwargs (dict): Other standard inputs for retinanet_mask.
 
+    Raises:
+        ValueError: temporal_ode not 'conv', 'lstm' or 'gru'
+
     Returns:
         tensorflow.keras.Model: Panoptic model with a backbone.
     """
     channel_axis = 1 if K.image_data_format() == 'channels_first' else -1
 
+    acceptable_modes = {'conv', 'lstm', 'gru'}
+    if temporal_mode.lower() not in acceptable_modes:
+        raise ValueError('Mode {} not supported. Please choose from {}.'.format(
+            temporal_mode.lower(), str(acceptable_modes)))
+

```

Comment:
more type assumptions that may fail.

Reply:
Is the assumption here that `temporal_mode` is a string?

Reply:
Yes